go.mod

```diff
@@ -82,3 +82,10 @@ replace (
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.18.5
 	k8s.io/sample-controller => k8s.io/sample-controller v0.18.5
 )
+
+require github.com/twelho/capi-existinginfra v0.0.0
+
+replace (
+	github.com/twelho/capi-existinginfra => ../capi-existinginfra
```

Comment:
this looks like it should have been removed

Reply:
Will be removed soon when I get the repo published and import paths updated.